lib/seahorn/Smt/TypeChecker.cc

```diff
@@ -37,6 +39,10 @@ class TCVR {
     if (isOp<ERROR_TY>(type)) {
       foundError(exp);
       m_errorMap.insert({exp, m_errorExp});
+    } else if (isOp<ERRORBINDER>(type)) {
```

Comment:
I would avoid calling an Error type `ERRORBINDER` if only to avoid confusion between `OpBind` and `OpBinder`. We use the root `Bind` too much in the code and it has already confused you in the past.

Reply:
I couldn't find a better name. I will give it some thought.